thirtyinch/src/test/java/net/grandcentrix/thirtyinch/test/TiTestPresenterTest.java

```diff
@@ -0,0 +1,68 @@
+package net.grandcentrix.thirtyinch.test;
+
+import static junit.framework.Assert.fail;
+import static org.assertj.core.api.Assertions.*;
+
+import android.support.annotation.NonNull;
+import net.grandcentrix.thirtyinch.TiPresenter;
+import net.grandcentrix.thirtyinch.TiView;
+import net.grandcentrix.thirtyinch.ViewAction;
+import org.junit.*;
+
+public class TiTestPresenterTest {
+
+    interface MockTiView extends TiView {
+
+        void helloWorld();
+
+    }
+
+    private TiPresenter<MockTiView> mockTiPresenter;
+
+    private MockTiView mockTiView;
+
+    @Before
+    public void setUp() throws Exception {
+        mockTiPresenter = new TiPresenter<MockTiView>() {
+
+            @Override
+            protected void onAttachView(@NonNull MockTiView view) {
+                super.onAttachView(view);
+
+                sendToView(new ViewAction<MockTiView>() {
+                    @Override
+                    public void call(MockTiView tiView) {
+                        tiView.helloWorld();
+                    }
+                });
+            }
+        };
+        mockTiPresenter.create();
+        mockTiView = new MockTiView() {
```

Comment:
use `Mockito.mock(MockTiView.class);` and verify that `helloWorld()` was called.